skiko/src/jvmMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.kt

```diff
@@ -28,8 +32,10 @@ internal class MetalRedrawer(
     private val windowHandle = layer.windowHandle
 
     private val frameDispatcher = FrameDispatcher(Dispatchers.Swing) {
-        update(System.nanoTime())
-        draw()
+        if (layer.isShowing) {
+            update(System.nanoTime())
```

Comment:
I just realized that this change conflicts with `awaitRedraw` method. `awaitRedraw` should draw into the native buffer, even if window is invisible.

Probably this change is okay, but `awaitRedraw` should be changed. I also realized why it was slow on macOs (because of the delay(300) when the window is occluded)

Let's discuss it next week.